src/DotNetProjectFile.Analyzers/Analyzers/MsBuild/ProvideCompliantPackageIcon.cs

```diff
@@ -0,0 +1,58 @@
+﻿
+namespace DotNetProjectFile.Analyzers.MsBuild;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+public sealed class ProvideCompliantPackageIcon() : MsBuildProjectFileAnalyzer(Rule.ProvideCompliantPackageIcon)
```

Comment:
Shouldn't this only be a requirement if the project is a package?

Reply:
Fair point. We do the same for the other package info related rules.